### ISSUE TITLE
Unserialize error in planer module

### ISSUE THREAD

**Issue by RoxyHalliwell** [jweiland-net/weather2#17]
For existing and new projects, we get the following error in the planer module when the "Call openweathermap.org api" task exists or is created:

```
#1588784142: Cannot unserialize TYPO3\CMS\Core\Log\Writer\AbstractWriter

BadMethodCallException thrown in file
... /typo3/sysext/core/Classes/Security/BlockSerializationTrait.php in line 38.
```

I guess the bug occurs since the latest patch of TYPO3 8.7.33.
We use PHP 7.3.10 and the latest extension version 2.0.3.

**Comment by froemken** [comment]
Hello @RoxyHalliwell 

thank you for reporting this issue. I can confirm this Bug. It has something to do with our logger-property in our task object, which was serialized for DB by TYPO3. But unserializing this logger object is not allowed by TYPO3 anymore.

I have just removed logger-property, so it was not serialized to DB anymore which works pretty fine. But currently you have to remove the old tasks and create it again. As I can't unserialize this string I can't change it, so no possibility to provide an UpgradeWizard. But maybe I can provide a deleting UpgradeWizard to delete all of our Tasks for you.

We will discuss that at the beginning of next week.

Or do you have any further ideas how to Upgrade these tasks?

Happy weekend

Stefan Frömken

**Comment by RoxyHalliwell** [comment]
Hey Stefan,

thank you for your reply. Unfortunately, I don't have any further ideas. An UpgradeWizard for deleting the tasks would be awesome, because I can't exactely figure out which task is which, can I? They look all the same in the database or is there trick to find out which one is the "Call openweathermap.org api"? Otherwise, I have to delete them one after another and check when the planer module works again. It's a bit of a nuisance.

Looking forward to your reply. Have a nice weekend, too.

**Comment by froemken** [comment]
Hello @RoxyHalliwell 

I have just released version 2.0.4 incl. an UpgradeWizard which modifies the serialized Tasks directly in DB (set Task-Logger to null).
Yeah...you must not delete all of our Tasks.

Have fun with the new version

Stefan
